### PR TITLE
Initial requirements document

### PR DESCRIPTION
**Pull request by ancorgs**
Just a starting point. The hard part starts now :-)

### CODE REVIEW

File: requirements.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
+# Requisites: 1:1 Management Tool(s) for ALP Host OS
+
+## General considerations
+
+The tool(s) should enable easy and convenient management of a so-called pet system (as opposed to
+cattle). Although SLE offers tools for configuring every aspect of a particular system (basically
+YaST and its myriad of modules), 1:1 management on ALP should be an exception rather than the rule.
+It can be assumed that systems that are part of either a data-center or cloud or edge infrastructure
+will be administered from a 1:many control system and only occasionally will be touched directly.
```
Comment (lslezak) [inline requirements.md:9]: NP: I'd use `1:M` term to be compatible with `1:1` used here.

File: requirements.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+
+The tool should focus on easing the management of a remote system (eg. the administrator connects
+to the system using a web browser) but it should also allow local administration (eg. using a
+keyboard and a screen connected to the system) when possible.
+
+Since ALP will be based on Tumbleweed, it will follow the same pattern of splitting the system
+configuration with a cascade approach like `/etc/example.conf` -> `/usr/etc/example.conf` ->
+`/usr/etc/example.d/*.conf` -> `/etc/example.d/*.conf`.
+
+The ALP Host OS will be an immutable transactional system. Most of the filesystem hierarchy will be
```
Comment (lslezak) [inline requirements.md:24]: "The ALP Host OS will be an immutable transactional system." Is that already a final decision? What about customers who prefer the old traditional RW approach?

**Review by joseivanlopez — APPROVED** [review]
LGTM

**Review by lslezak — COMMENTED** [review]
Few minor notes.

File: requirements.md (hunk 2 above)
Comment (ancorgs) [reply requirements.md:24]: Yes, that's a final decision as far as I can tell.

**Pull request merged**